share/qt/Info.plist.in

```diff
@@ -17,10 +17,10 @@
   <string>APPL</string>
 
   <key>CFBundleGetInfoString</key>
-  <string>@CLIENT_VERSION_MAJOR@.@CLIENT_VERSION_MINOR@.@CLIENT_VERSION_REVISION@.@CLIENT_VERSION_BUILD@, Copyright © 2009-@COPYRIGHT_YEAR@ @COPYRIGHT_HOLDERS_FINAL@</string>
```

Comment:
Removing the version from here will mean it doesn't show up in the `Version` portion of the app info. I don't think that's what we want to do.
![versions](https://user-images.githubusercontent.com/863730/68948058-10ac6600-0785-11ea-8bf2-86fb5c1393a1.png)